### ISSUE TITLE
API kulcs a kodban

### ISSUE THREAD

**Issue by MagyarMate** [SzombatiAttila/stock_predictor#1]
https://github.com/SzombatiAttila/stock_predictor/blob/cc15ebecadeb98620ff4ce39e77f689237d7cd19/scraper/reddit_official_api.py#L9

Cso Attila, remelem jol vagy :) API kulcsokat, jelszvakat vedd ki a kodbol mielott feltoltod githubra/barhova, masik otlet, hogy mysql containernek csinalj egy volumeot a hoston.

Udv,
MM

**Comment by SzombatiAttila** [comment]
Hali, koszi az eszrevetelt! A reddit scraper meg nem 100%-os igy elsiklottam felette. :) A docker volume alatt arra celzol hogy ki legyenek mentve valamilyen formaban a tablak?

**Comment by MagyarMate** [comment]
Igen, mert akkor megmarad. Igy ha ujra krealod a containert lepucolja a database-t.

**Comment by attilaszombati** [comment]
most in memory db-re switcheltem, ez kezelhetobb
